mypy/messages.py

```diff
@@ -247,6 +247,8 @@ def format_simple(self, typ: Type, verbosity: int = 0) -> str:
             return 'None'
         elif isinstance(typ, AnyType):
             return '"Any"'
+        elif isinstance(typ, DeletedType):
+            return '<Deleted>'
```

Comment:
Minor suggestion: Maybe reword this as `<deleted>` (without capital 'D') as otherwise this may look like `Deleted` is a user-level concept, similar to `Any` for example, even though it's an internal thing only.